### PR TITLE
feat: Support for blend mode arg in human readable format

### PR DESCRIPTION
**Pull request by kgoedecke**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: utils/maps.js [hunk 1]
```diff
@@ -64,9 +64,39 @@ Object.keys(verticalAlignmentMap).forEach(key => {
...
+  difference: 10,
+  exclusion: 11,
+  hue: 12,
+  saturation: 13,
+  luminosity: 14,
+  plusDarker: 15,
+  plusLighter: 16,
+};
+
+Object.keys(blendModeMap).forEach(key => {
```
Comment (dbanksdesign) [inline utils/maps.js]: Clever!

File: models/GraphicsContextSettings/GraphicsContextSettings.js [hunk 2]
```diff
@@ -37,6 +39,9 @@ class GraphicsContextSettings {
       Object.assign(this, json);
     } else {
       Object.assign(this, GraphicsContextSettings.Model, args);
+      Object.assign(this, {
```
Comment (dbanksdesign) [inline models/GraphicsContextSettings/GraphicsContextSettings.js]: I think without the `GraphicsContextSettings.Model` this won't get the `_class` attribute.

Comment (kgoedecke) [reply models/GraphicsContextSettings/GraphicsContextSettings.js]: Line 41 (https://github.com/amzn/sketch-constructor/pull/67/files#diff-9147adf669b4522b117eacb792424815R41) still assigns the Model, or am I missing something? The we just overwrite the blendMode with the values from the map.

Comment (kgoedecke) [reply models/GraphicsContextSettings/GraphicsContextSettings.js]: Oh duh... will do that, thanks for the hint.

Comment (kgoedecke) [reply models/GraphicsContextSettings/GraphicsContextSettings.js]: @dbanksdesign done :)

**Pull request merged**